biogtr/models/transformer.py

```diff
@@ -179,8 +187,23 @@ def forward(self, frames: list[Frame], query_frame: int = None):
 
         emb = emb.permute(1, 0, 2)  # (total_instances, batch_size, embed_dim)
 
-        query_inds = None
+        batch_size, total_instances, embed_dim = reid_features.shape
+
+        reid_features = reid_features.permute(
+            1, 0, 2
+        )  # (total_instances x batch_size x embed_dim)
```

Comment:
Use standard shape tuple syntax for documenting shapes:

```suggestion
        )  # (total_instances, batch_size, embed_dim)
```